jwplayer-appletv-web-app/js/playlist/PlaylistLoader.js

```diff
@@ -89,7 +89,8 @@ var PlaylistLoader = function() {
         mediaItem.mediaid = playlistItem.mediaid;
         mediaItem.description = playlistItem.description;
         mediaItem.title = playlistItem.title;
-        mediaItem.artworkImageURL = _checkScheme(playlistItem.image);
+        mediaItem.artworkImageURL = playlistItem.image ?
+          _checkScheme(playlistItem.image) : "";
```

Comment:
@RikHeijdens I think image should be sufficient for now.  We already handle empty descriptions and titles gracefully, right?